Gaijinizer patch/Map057.txt

```diff
@@ -17,9 +17,9 @@ Yuuma.#MTLed
 > CONTEXT: Map057/events/5/pages/0/20/Dialogue/yuuma
 > CONTEXT: Map057/events/5/pages/1/21/Dialogue/yuuma
 > CONTEXT: Map057/events/5/pages/2/20/Dialogue/yuuma
-There is a little more. The park is two blocks
-away. If you follow the back of the park along
-the way, you will see the harbor.#MTLed
+Just a little more, the park's two blocks away.
+If you keep following the back of the park, you'll
+see the harbor soon enough.#Ron
```

Comment:
A tiny bit of flow change to the second part:
>_**If you keep following the road behind the park,
you'll see the harbor soon enough.**_